.golangci.yml

```
@@ -0,0 +1,27 @@
+run:
+  timeout: "10m"
+linters:
+  disable-all: true
+  enable:
+    # Format
+    - "gofumpt" # Gofumpt checks whether code was gofumpt-ed. Which is a superset of gofmt that handles some cases that gofmt missed.
+    - "gci" # Gci controls Go package import order and makes it always deterministic.
+    - "asciicheck" # Checks that all code identifiers does not have non-ASCII symbols in the name.
+    - "bidichk" # Checks for dangerous unicode character sequences.
+    # Error
+    - "errcheck"
+    # Simplify
+    - "gosimple"
+    - "govet"
+    - "ineffassign"
+    - "staticcheck"
+    - "unused"
+linters-settings:
+  gci:
+    sections:
+      - standard
+      - default
+      - prefix(github.com/pb33f/libopenapi-validator)
+      - blank
```

There is a problem according with this configuration according to me

See [here](https://github.com/pb33f/libopenapi-validator/pull/101#discussion_r1832526086) or [here](https://github.com/pb33f/libopenapi-validator/pull/101#discussion_r1832537252)

There are two modules under pb33f, and the current gci config won't address it fully

Here is what I would recommend

```suggestion
      - default
      - prefix(github.com/pb33f/libopenapi-validator,github.com/pb33f/libopenapi)
      - blank
```

Or
```suggestion
      - default
      - prefix(github.com/pb33f)
      - blank
```

But maybe this one will work, it's worth checking

```suggestion
      - default
      - localmodule
      - blank
```

If you disagree to group them, you can use this maybe

```suggestion
      - default
      - prefix(github.com/pb33f/libopenapi-validator)
      - prefix(github.com/pb33f/libopenapi)
      - blank
```